src/Serializer/JsonEncoder.php

```diff
@@ -34,7 +34,8 @@ public function __construct(string $format, BaseJsonEncoder $jsonEncoder = null)
 
         // Encode <, >, ', &, and " for RFC4627-compliant JSON, which may also be embedded into HTML.
         $this->jsonEncoder = $jsonEncoder ?: new BaseJsonEncoder(
-            new JsonEncode(JsonResponse::DEFAULT_ENCODING_OPTIONS), new JsonDecode(true)
+        // The value "15" will be replaced by  "JsonResponse::DEFAULT_ENCODING_OPTIONS" when Symfony HttpFoundation 2.7 support will be dropped
```

Comment:
Why not use the PHP constants directly? `JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP | JSON_HEX_QUOT`

We do not necessarily want to stick to `JsonResponse::DEFAULT_ENCODING_OPTIONS`. For example, I think `JSON_UNESCAPED_UNICODE` should be the default in this day and age. And we can remove one unused import too...